cypress/integration/invoices/invoices.testPolSearchPlugin.spec.js

```diff
@@ -0,0 +1,74 @@
+import TopMenu from '../../support/fragments/topMenu';
+import NewInvoice from '../../support/fragments/invoices/newInvoice';
+import Invoices from '../../support/fragments/invoices/invoices';
+import testType from '../../support/dictionary/testTypes';
+import VendorAddress from '../../support/fragments/invoices/vendorAddress';
+import newOrder from '../../support/fragments/orders/newOrder';
+import newOrderLine from '../../support/fragments/orders/newOrderLine';
+import Orders from '../../support/fragments/orders/orders';
+
+describe('ui-invoices: test POL search plugin', () => {
+  const invoice = { ...NewInvoice.defaultUiInvoice };
+  const vendorPrimaryAddress = { ...VendorAddress.vendorAddress };
+  const order = { ...newOrder.defaultOrder };
+  const orderLine = { ...newOrderLine.defaultOrderLine };
+  const locationName = 'Main Library';
+
+  before(() => {
+    cy.getToken(Cypress.env('diku_login'), Cypress.env('diku_password'));
+    cy.getOrganizationApi({ query: `name=${invoice.vendorName}` })
+      .then(organization => {
+        invoice.accountingCode = organization.erpCode;
+        Object.assign(vendorPrimaryAddress,
+          organization.addresses.find(address => address.isPrimary === true));
+        order.vendor = organization.id;
```

Comment:
'vendor' is API parameter needed for order creation and organization entity is basically the same, as you can see e.g. in couple lines of code below we use organization.id for some other parameters, not even 'vendor'. Don't see reason to rename it in autotests since we're following approach with different naming in implementation itself.